test/client/client-test.js

```diff
@@ -1,86 +1,92 @@
 const expect = require('chai').expect
+const Client = require('../../client')
 
 const API_TOKEN = process.env.API_TOKEN
 
-const config = {
-  host: 'https://service-server-production.livingdocs.io',
-  apiToken: API_TOKEN,
-  basePath: '/api/v1'
+const defaultConfig = {
+  url: 'https://server.livingdocs.io',
+  accessToken: API_TOKEN
 }
 
-function createClient () {
-  return require('../../client')(config)
-}
+const createClient = config => new Client(config)
 
-describe('client', function () {
-  it('should be an object', function () {
-    const client = createClient()
+describe('client', () => {
```

Comment:
As you mentioned in the PR description in mocha you should always use `function` for `describe` and `it` methods.

Reply:
It's a mocha convention to use `this`  in `before` and `beforeEach` blocks to isolate variables per test run. And for `this` to work you need `functions`.